bots/tests-invoke

```diff
@@ -373,8 +373,13 @@ class PullTask(object):
 
 
         env = os.environ.copy()
-        if "TEST_DATA" in env:
-            del env["TEST_DATA"]
+        # we can only use the cache for master; branches still use the old
+        # test/images/ symlink farm which we must not destroy
+        if opts.rebase:
```

Comment:
Actually, why not check it directly with `git ls-tree -d HEAD:bots/images` - that also works after the force-checkout of `bots/` from master, as we don't commit that.